Bottom: cbb03a215719cf0f596a455dabbd38a83942afb1
Top:    b7e6045adfa2d7e97c19eb30ae9314787544585a
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-02-15 20:16:18 +0000

Refresh of 0002-Fix-poly-contours.patch

---

diff --git a/src/polygon1.c b/src/polygon1.c
index f0b70ef..fef7087 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -1703,9 +1703,8 @@ Collect (char poly, jmp_buf * e, PLINE * a, POLYAREA ** contours, PLINE ** holes
   VNODE *cur;
   DIRECTION dir = UNINITIALISED;
 
-  cur = (&a->head)->next->next->next->next->next; /* Breaks circ_seg_test9.pcb */
-//  cur = (&a->head); //    ->next->next->next->next->next; /* Breaks circ_seg_test9.pcb */
-//  cur = (&a->head)        ;
+  cur = (&a->head)        ;
+//  cur = (&a->head)->next->next->next->next->next; /* Breaks circ_seg_test9.pcb */
   do
     {
 #if 0
